
Include the __aligned_* from 3.2 include/types.h to fix build.

Signed-off-by: Thomas Backlund <tmb@mageia.org>


--- iptables-1.4.12.2/include/xtables.h.in.orig	2012-01-02 19:19:09.000000000 +0200
+++ iptables-1.4.12.2/include/xtables.h.in	2012-01-06 16:22:57.900569755 +0200
@@ -34,6 +34,11 @@
 #define XTABLES_VERSION "libxtables.so.@libxtables_vmajor@"
 #define XTABLES_VERSION_CODE @libxtables_vmajor@
 
+/* copied from linux 3.2 include/types.h to fix build */
+#define __aligned_u64 __u64 __attribute__((aligned(8)))
+#define __aligned_be64 __be64 __attribute__((aligned(8)))
+#define __aligned_le64 __le64 __attribute__((aligned(8)))
+
 struct in_addr;
 
 /*
