diff -up iptables-1.8.7/iptables/ip6tables.c.1~ iptables-1.8.7/iptables/ip6tables.c
--- iptables-1.8.7/iptables/ip6tables.c.1~	2021-01-16 18:19:34.744528527 +0100
+++ iptables-1.8.7/iptables/ip6tables.c	2021-01-16 18:20:15.278208340 +0100
@@ -44,6 +44,7 @@
 #include <sys/socket.h>
 #include "ip6tables-multi.h"
 #include "xshared.h"
+#include "libiptc/libiptc.h"
 
 static const char unsupported_rev[] = " [unsupported revision]";
 
